src/SharedFixerConfig.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer;
+
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+final class SharedFixerConfig
```

Comment:
Question:
should we make `SharedFixerConfig ` or `WhitespacesFixerConfig` ?
(mimic the potential chaneg to `...AwareInterface`)